resources/automagic_dashboards.yaml

```diff
@@ -0,0 +1,26 @@
+automagicDashboards:
```

Comment:
Is this being loaded in anywhere? It doesn't seem to be valid yaml.

Reply:
No, this is some example or something that's long past its use. I'll remove it.